gh-pages/docs/description/array_ephys.md

```diff
@@ -0,0 +1,67 @@
+# Array Electrophysiology Element
```

Comment:
It looked odd to me that the drop-down was inconsistent in the use of `Element` - So I removed from respective titles. Will that be reflected in the dropdown menu when rendered?
```suggestion
# Array Electrophysiology
```

Reply:
Thanks @CBroz1.  These changes should be reflected in the dropdown menu.